update/Pos/Update/Poll/RollTrans.hs

```diff
@@ -77,6 +77,8 @@ instance MonadPoll m => MonadPoll (RollT m) where
         insertIfNotExist id unChangedPropsL getProposal
         deactivateProposal id
 
+    setSlottingData = lift . setSlottingData
```

Comment:
He-he, it could be bug :)

Better ask someone when you such suspicious places. I am not sure whether my intent was to not write this definition because I want it to look like this.
(maybe it didn't look suspicious to you, dunno)

Anyway I will check whether this logic is correct.